### ISSUE TITLE
Example project

### ISSUE THREAD

**Issue by mhaagens** [Ephem/react#3]
Any chance of an example project with minimal setup?

**Comment by Ephem** [comment]
Absolutely, it's in the pipes!

I'm trying to figure out how to best demo this, seeing how most parts of such an example would really demo Suspense patterns on the server side rather than this library itself. Those patterns doesn't quite exist yet for obvious reasons, but I will definitely create some kind of minimal example. :)

**Comment by Ephem** [comment]
@mhaagens I have added a [Redux-example](https://github.com/Ephem/react-lightyear/tree/lightyear/examples/redux).

I decided to do it with Redux since it's something many have experience with. I plan to add more examples in the future, including one with some kind of barebones cache-implementation, but hopefully this is useful meanwhile? I would love suggestions for improvements!

It might be interesting to read the implementation note in the example readme, it explains why it is not currently possible to use `connect`. 😄
